tests/kafkatest/tests/streams/streams_broker_compatibility_test.py

```diff
@@ -60,17 +53,15 @@ def __init__(self, test_context):
                                            self.output,
                                            "stream-broker-compatibility-verify-consumer")
 
-    def setUp(self):
-        self.zk.start()
-
 
     @cluster(num_nodes=4)
-    @matrix(broker_version=[str(LATEST_1_0),str(LATEST_1_1),str(LATEST_2_0),str(LATEST_2_1),
-                            str(LATEST_2_2),str(LATEST_2_3),str(LATEST_2_4),str(LATEST_2_5),
-                            str(LATEST_2_6),str(LATEST_2_7),str(LATEST_2_8),str(LATEST_3_0),
-                            str(LATEST_3_1),str(LATEST_3_2),str(LATEST_3_3),str(LATEST_3_4),
-                            str(LATEST_3_5),str(LATEST_3_6),str(LATEST_3_7),str(LATEST_3_8)])
-    def test_compatible_brokers_eos_disabled(self, broker_version):
+    @matrix(broker_version=[str(LATEST_2_8),
```

Comment:
2.8.2 does not support Producer IDs in kraft (see https://github.com/apache/kafka/commit/e97cff2702b6ba836c7925caa36ab18066a7c95d).

Reply:
ack